plugins/wazo-gigaset/common/common.py

```diff
@@ -195,6 +203,13 @@ def _add_phonebook(self, raw_config):
         uuid_format = '{scheme}://{hostname}:{port}/0.1/directories/lookup/default/gigaset/{user_uuid}?'  # noqa: E501
         plugins.add_xivo_phonebook_url_from_format(raw_config, uuid_format)
 
+    def _add_country_and_lang(self, raw_config):
```

Comment:
Same as the comment above. It would be more appropriate to call it country tone than country and language.

Reply:
Done.